_api-reference/ingest-apis/simulate-ingest.md

```diff
@@ -36,48 +34,54 @@ POST /_ingest/pipeline/35678/_simulate
 ```
 {% include copy-curl.html %}
 
-## Path and HTTP methods
+## Example requests
 
-Simulate the last ingest pipeline created.
+Requests to simulate the lastest ingest pipeline created:
 
 ```
 GET _ingest/pipeline/_simulate
 POST _ingest/pipeline/_simulate
 ```
 
-Simulate a single pipeline based on the pipeline's ID.
+Requests to simulate a single pipeline based on the pipeline identifier.
 
 ```
-GET _ingest/pipeline/{id}/_simulate
-POST _ingest/pipeline/{id}/_simulate
+GET _ingest/pipeline/pipeline-id/_simulate
+POST _ingest/pipeline/pipeline-id/_simulate
 ```
 
-## URL parameters
+## Path parameters
+
+Parameter | Type | Description
+:--- | :--- | :---
+`pipeline` | String | Pipeline identifier or wildcard expression of pipeline identifiers used to limit the request. 
 
-All URL parameters are optional.
+## Query parameters 
+
+Query parameters are optional.
 
 Parameter | Type | Description
 :--- | :--- | :---
-verbose | boolean | Verbose mode. Display data output for each processor in executed pipeline.
+`verbose` | Boolean | Verbose mode. Display data output for each processor in the executed pipeline.
 
 ## Request body fields
 
-Field | Required | Type | Description
-:--- | :--- | :--- | :---
-`pipeline` | Optional | object | The pipeline you want to simulate. When included without the pipeline `{id}` inside the request path, the response simulates the last pipeline created.
-`docs` | Required | array of objects | The documents you want to use to test the pipeline.
+Field | Type | Description
+:--- | :--- | :---
+`pipeline` | Object | The pipeline to be simulated. If the pipeline identifier is not included, then the response simulates the lastest pipeline created. Optional.
+`docs` | Array | The documents to be used to test the pipeline. Required.
 
 The `docs` field can include the following subfields:
 
-Field | Required | Type | Description
+Field | Type | Description
 :--- | :--- | :---
-`id` | Optional |string | An optional identifier for the document. The identifier cannot be used elsewhere in the index.
-`index` | Optional | string | The index where the document's transformed data appears.
-`source` | Required | object | The document's JSON body.
+`id` | String | A unique identifier for a document. The identifier cannot be used elsewhere in the index. Optional.
+`index` | String | A collection of documents that have undergone specfic data transformation within an ingest pipeline. Optional.
```

Comment:
Not all of the documents in the index might have undergone the data transformation. The index is only to tell where this document will get ingested.

```suggestion
`index` | String | The index where the document's transformed data appears. Optional.
```